### PR TITLE
auto OCP-22640 Add HTPasswd IDP on console

### PR DESCRIPTION
**Pull request by yapei**
@xiaocwan   FYI

### CODE REVIEW

File: features/step_definitions/admin_console.rb [hunk 1]
```diff
@@ -8,9 +8,9 @@
 
   version = env.webconsole_executor.get_master_version(user, via_rest: true)
   step "I have a browser with:", table(%{
+    | rules        | lib/rules/web/admin_console/#{version}/  |
```
Comment (yapei) [inline features/step_definitions/admin_console.rb]: changes in #320

**Comment by yapei** [comment]
@xiaocwan  @SSshahan  Ready for review, PTAL thanks

**Comment by SSshahan** [comment]
LGTM. Thanks

**Comment by xiaocwan** [comment]
LGTM. Thanks

**Comment by yapei** [comment]
@akostadinov  @pruan-rht  Can you help merge if no objections?

**Pull request merged**